build-system/release-workflows/promote-job.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const fs = require('fs-extra');
+const {createPullRequest} = require('octokit-plugin-create-pull-request');
```

Comment:
A plugin just for octokit types? 😝  Can we use https://www.npmjs.com/package/@octokit/types instead?

Reply:
That's not a type, it's a plugin that wraps multiple steps into a single function - to create a PR *and* modify files in it